apps/bedrock-content-generator/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "bedrock-content-generator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@aws-sdk/client-bedrock": "^3.474.0",
+    "@aws-sdk/client-bedrock-runtime": "^3.474.0",
+    "@contentful/app-sdk": "4.23.0",
+    "@contentful/f36-components": "4.45.0",
+    "@contentful/f36-icons": "^4.27.0",
+    "@contentful/f36-tokens": "4.0.2",
+    "@contentful/react-apps-toolkit": "1.2.16",
+    "@contentful/rich-text-plain-text-renderer": "^16.0.5",
+    "@emotion/react": "^11.11.1",
+    "@segment/analytics-next": "^1.55.0",
+    "contentful-management": "10.28.1",
+    "launchdarkly-react-client-sdk": "^3.0.8",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "usehooks-ts": "^2.9.1"
+  },
+  "scripts": {
+    "start": "vite",
+    "start:dev": "NODE_ENV='development' && npm start",
+    "start:preview": "npm run build && vite preview",
+    "start:staging": "NODE_ENV='staging' && npm run start:preview",
+    "start:prod": "NODE_ENV='production' && npm run start:preview",
+    "build": "tsc && vite build --mode ${NODE_ENV:='production'}",
+    "test": "vitest --logHeapUsage --coverage",
+    "test:ci": "vitest",
+    "lint": "eslint src --max-warnings 0",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEFINITIONS_ORG_ID} --definition-id 3iszK8Gl7aaxLvxGyCOhgA --token ${CONTENTFUL_CMA_TOKEN}",
+    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEV_TESTING_ORG_ID} --definition-id 7sjBFUzBxj1fRrI0eUBwLc --token ${TEST_CMA_TOKEN}"
```

Same as above, we can use this one for the `staging` deploy which remains names as `deploy:test`

```suggestion
    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEV_TESTING_ORG_ID} --definition-id 63AGu5jLZaruxYGLmPrWSm --token ${TEST_CMA_TOKEN}"
```